lldb/include/lldb/Symbol/Type.h

```diff
@@ -34,7 +34,7 @@ struct CompilerContext {
   }
   bool operator!=(const CompilerContext &rhs) const { return !(*this == rhs); }
 
-  void Dump() const;
+  void Dump(Stream *s) const;
```

Comment:
I think Alex Langford is trying to get rid of functions that take "Stream *s" and replace them with "Stream &s"

Reply:
Yes, we basically always want the stream to be a valid pointer, a reference makes more sense here.

Reply:
Sounds good to me!